src/js/views/patients/sidebar/action-sidebar.hbs

```diff
@@ -19,4 +19,9 @@
   <h3>Activity Log</h3>
   <div data-activity-region></div>
 </div>
+<hr>
```

Comment:
no hr we'll add that to the class `action-sidebar__timestamps`

Reply:
is that class correct?